model/pyproject.toml

```diff
@@ -15,8 +15,26 @@ polars = "^0.20.13"
 matplotlib = "^3.8.3"
 pillow = "^10.3.0" # See #56 on CDCgov/multisignal-epi-inference
 ipykernel = "^6.29.3"
+numpydoc = "^1.6.0"
```

Comment:
Should this go in the `[tool.poetry.group.test.dependencies]` section? Or is it needed to run the models?

Reply:
I wonder if this should be required for the project, @AFg6K7h4fhy2. Since the package can still run without `numpydoc`. The same applies to the `ipykernel` (which I added). What are your thoughts, @dylanhmorris?

Reply:
Presumably `[tool.poetry.group.docs.dependencies]`, no?
https://python-poetry.org/docs/managing-dependencies/#optional-groups

Reply:
Addressed in 0710c3a